interface/fax/fax_dispatch.php

```diff
@@ -846,7 +839,11 @@ function allCheckboxes(issel) {
 ?>
 
 </table>
-</center>
 </form>
+<script>
+    $(function () {
+        $('[data-toggle="tooltip"]').tooltip();
+    });
+</script>
```

Comment:
Unable to fully test this script, but looks good. @stephenwaite , are you able to test this script (I wasn't able to every see any pages to choose on the dispatch gui when clicked on it)

Reply:
wish we could `apk add hylafax` :)